src/Objs/SearchResults/SearchInput.js

```diff
@@ -11,12 +11,10 @@ class SearchInput extends React.Component {
   }
 
   static getDerivedStateFromProps(nextProps, prevState) {
-    if (!prevState.params || !nextProps.params || prevState.params.q === nextProps.params.q) {
-      // Return null to indicate no change to state.
-      return null;
-    }
+    const newQ = (nextProps.params || {}).q;
```

Comment:
FYI could also be written as
`const newQ = nextProps.params && nextProps.params.q;`
or even
`const q = nextProps.params && nextProps.params.q;`